doc/manuals/user/filtering.md

```diff
@@ -136,3 +137,35 @@ operation filter.
 
 [Top](#top)
 
+## String filter
+
+The scope corresponding to this type is "FIWARE::StringFilter".
+
+      ...
+      {
+          "restriction": {
+              "scopes": [
+                  {
+                      "type": "FIWARE::StringFilter",
+                      "value": "temperature<24;humidity==75..90;status=running"
+                  }
+              ]
+          }
+      }
+      ...
+
+This scope allows to express filtering conditions such as equality, unequality,
+greater/less than, range or existence.
+
+There isn't any URL parameter correspondence for this filter in NGSI v1. In NGSI v2
+it correspons with the `q` parameter:
+
+    curl localhost:1026/v2/entities?q=temperature<24;humidity==75..90;status=running
```

Comment:
This example should be modified, as it wont work ... (adding quotes fixes the problem)

    curl 'localhost:1026/v2/entities?q=temperature<24;humidity==75..90;status=running'